.github/actions/javascript/checkDeployBlockers/index.js

```diff
@@ -305,6 +307,7 @@ class GithubUtils {
         accessiblePRList = [],
         deployBlockers = [],
         resolvedDeployBlockers = [],
+        resolvedInternalQAPRs = [],
```

Comment:
I agree and I must admit that when I was trying to look into this it took me a while to orient myself in the code, probably breaking it up into different parameters would be good. I would leave this for another clean up once we will need to make changes to this code to avoid breaking this even more haha

Reply:
@roryabraham I am not against doing it sooner than later, but I would say in a new issue and PR so this internalQA bug is fixed. 

I will be happy to work on the refactoring, but got some other stuff lined up right so I am not sure how soon I could get that coverd and we dont want this bug to annoy deployer for much longer 😄

Reply:
@roryabraham Created the issue here https://github.com/Expensify/App/issues/9999

Reply:
Thanks for creating the issue. Pretty cool issue number #9999 too 🙂